src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,107 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
 import './NewCommentForm.scss';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+type Props = {
+  selectedPostId: number,
+  onAddNewComment: CallableFunction,
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  selectedPostId,
+  onAddNewComment,
+}) => {
+  const [nameValue, setNameValue] = useState<string>('');
+  const [emailValue, setEmailValue] = useState<string>('');
+  const [commentValue, setCommentValue] = useState<string>('');
+  const [isSubmitError, setIsSubmitError] = useState<boolean>(false);
+
+  const clearCommentForm = () => {
+    setNameValue('');
+    setEmailValue('');
+    setCommentValue('');
+  };
+
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+
+    if (nameValue === '' || emailValue === '' || commentValue === '') {
```

Comment:
you could pass it into separate variable and check like `!nameValue`

Reply:
done